src/domain/places/placeList/PlaceList.tsx

```diff
@@ -100,7 +100,7 @@ const PlaceListContainer: React.FC = () => {
   const { count, onSearchSubmit, ...listProps } = useCommonListProps({
     defaultSort: DEFAULT_PLACE_SORT,
     listId: placeListId,
-    meta: placesData?.places.meta,
+    meta: placesData?.places?.meta,
```

Comment:
This seems a bit unrelated to the price group changes so perhaps could be in its own commit?